caffe2/contrib/tensorrt/tensorrt_op_trt.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include "caffe2/contrib/tensorrt/trt_utils.h"
+#include "caffe2/core/context_gpu.h"
+#include "caffe2/core/operator.h"
+
+#include <NvInfer.h>
+#include <iostream>
+#include <memory>
```

Comment:
`iostream` and `memory` doesn't seem needed for the interface file

Reply:
I'll remove them and see. :)